src/features/basic/show-space-remaining.ts

```diff
@@ -0,0 +1,68 @@
+import { shipsStore } from '@src/infrastructure/prun-api/data/ships';
+import { storagesStore } from '@src/infrastructure/prun-api/data/storage';
+import { watchEffectWhileNodeAlive } from '@src/utils/watch';
+
+function formatFixed(f: number) {
+  const x = f % 1 ? f.toFixed(2) : f;
+  return x.toString().replace(/\B(?<!\.\d*)(?=(\d{3})+(?!\d))/g, ',');
+}
+
+function getStore(tile: PrunTile) {
+  const storeId =
+    tile.command.startsWith('SHPI') && tile.parameter
+      ? shipsStore.getByRegistration(tile.parameter)?.idShipStore
+      : tile.parameter;
+  if (!storeId) {
+    return;
+  }
+  return storagesStore.getById(storeId);
+}
+
+function onTileReady(tile: PrunTile) {
+  subscribe($$(tile.anchor, C.StoreView.column), column => {
+    const capacities = _$$(column, C.StoreView.capacity);
+    if (capacities.length < 2) {
+      return;
+    }
+    const weightIndex = capacities.findIndex(x => x.textContent?.includes('Weight'));
+    const volumeIndex = capacities.findIndex(x => x.textContent?.includes('Volume'));
+
+    if (weightIndex === -1 || volumeIndex === -1) {
+      return;
+    }
+    const weightBar = capacities[weightIndex].getElementsByTagName('progress')[0];
+    const volumeBar = capacities[volumeIndex].getElementsByTagName('progress')[0];
+    if (!weightBar || !volumeBar) {
+      return;
+    }
+
+    watchEffectWhileNodeAlive(weightBar, () => {
+      const store = getStore(tile);
+      if (!store) {
+        return;
+      }
+      const weightAvailable = formatFixed(store.weightCapacity - store.weightLoad);
+      (capacities[weightIndex].lastChild as Text).textContent = ` (${weightAvailable}t)`;
+    });
```

Comment:
To avoid issues if molp changes the node structure inside the `C.StoreView.capacity` element, append a new element instead of modifying an existing one.
```suggestion
    const weightText = computed(() => {
      const store = getStore(tile);
      return store.value ? ` (${formatFixed(store.weightCapacity - store.weightLoad)})` : undefined;
    });
    const weightSpan = createReactiveSpan(capacities[weightIndex], weightText);
    weightSpan.style.whiteSpace = 'pre';
    capacities[weightIndex].appendChild(weightSpan);
```